app/api/v1/endpoints/blog.py

```diff
@@ -22,7 +22,7 @@ def create_blog_endpoint(
     """
     Create a new blog.
     """
-    return create_blog(db, blog_in)
+    return create_blog(db, blog_in, author=current_user)
```

Comment:
Add try exceptions and handle common errors with error message use Response.build_failed_response custom class method.